lib/lambda/getPackageActions.ts

```diff
@@ -35,11 +37,6 @@ export const getPackageActions = async (event: APIGatewayEvent) => {
         body: { message: "Not authorized to view resources from this state" },
       });
 
-    if (!result.found)
```

Comment:
removed this because it's checking the same things as line 25 and giving the same response. Also, you should do you result not found checks before doing other stuff to avoid unnecessary calls afterwards, like the auth check on line 32.